HT5.py

```diff
@@ -0,0 +1,111 @@
+import datetime
+import random
+
+
+class Publication:
+    def __init__(self, name):
+        self.name = name

```

Comment:
I think it is more reasonable to have parameters of the entity here, like "title", "body".
And then you can extend child classes with "specific_parameter" like expiration date or smth

Reply:
Ok. Just want to emphasize that lector used "name" in the example and I can't read thoughts